src/app/shared/vote-form/vote-form.component.ts

```diff
@@ -81,23 +82,17 @@ export class VoteFormComponent implements OnInit {
   }
 
   stageExistingVotes() {
-    const votesObservable = this.ers.listVote({ position: this.position.id });
-    votesObservable.subscribe(
-      (data) => {
-        const existingVotes = data;
-        for (const vote of existingVotes) {
-          this.stageVote(vote);
-        }
-      }, (err) => {
-      }, () => {
-        // disable cast votes button if stagedVotes array length is 0
-        if (this.stagedVotes.length === 0) {
-          this.alertUser = true;
-        } else {
-          this.alertUser = false;
-        }
+    for (const vote of this.votes) {
+      if (this.position.id === vote.position) {
+        this.stageVote(vote);
       }
-    );
+    }
```

Comment:
@kadensu and @nick97eagles I did a merge conflict resolution between your two changes, I think I did it right but just wanted to let you guys know in case you notice anything wrong with this. I'm running into errors while testing but haven't ruled out my local database as the issue.